packages/web-components/tsconfig.json

```diff
@@ -11,6 +11,7 @@
     "target": "es2015",
     "module": "ESNext",
     "allowJs": true,
+    "skipLibCheck": true,
```

Comment:
Don't love this, but I won't block. We should figure out what's causing this to throw long-term. Thanks for digging into the upgrade!

Reply:
Yeah, it's more an alignment thing. All of the other packages use the same setting. If we move away from it, it'll be a repo wide change.